frontend/src/components/DownloadModal.tsx

```diff
@@ -29,7 +29,10 @@ export function DownloadModal({
     <Modal show={true} size={"sm"}>
       <Modal.Body>
         <div className="d-flex flex-column align-items-center">
-          <p>Preparing to download data...</p>
+          <p>
+            Downloading the latest version of your data request and converting
+            it to TSV format. This may take up to 30 seconds.
+          </p>
```

Comment:
Ben asked to clarify the export duration for bigger exports. That way, users won't feel like something went wrong when waiting too long.